src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+public class Task {
+    private String name;
+    private boolean isDone;
+
+    private static int tasksCount = 0;
+    private static Task[] tasks = new Task[100];
+
+    Task(String name){
+        this.name = name;
+        isDone = false;
+    }
+
+    public static void listTasks(){
+        System.out.println("-----------------------------------");
+        for (int i = 0; i < tasksCount; i++){
+            System.out.print((i + 1) + ".");
+            System.out.print("[");
+            if (tasks[i].isDone){
+                System.out.print("X");
+            } else {
+                System.out.print(" ");
+            }
+            System.out.print("] ");
+            System.out.println(tasks[i].name);
+        }
+        System.out.println("-----------------------------------");
+    }
+    public static void addTask(String taskName){
+        System.out.println("-----------------------------------");
+        System.out.println("added: " + taskName);
+        tasks[tasksCount] = new Task(taskName);
+        tasksCount++;
+        System.out.println("-----------------------------------");
+    }
+
+    public static void response(String command){
```

Comment:
"command" is a slightly ambiguous name especially as it is a string of multiple words, maybe consider changing it to a clearer name